data/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.0.4",
+  "version": "2.0.5",
```

Comment:
No need to bump the manifest in this commit. We should bump it together with the version number in package.json when a new release is created.

Reply:
Should be fixed now.